libs/components/action-bars/src/lib/modules/summary-action-bar/summary-action-bar-adapter.service.ts

```diff
@@ -22,6 +22,14 @@ export class SkySummaryActionBarAdapterService {
     this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
   }
 
+  public focusChevron(chevronElement: ElementRef) {
+    /* sanity-check */
+    /* istanbul ignore else */
+    if (chevronElement) {
```

Comment:
In what scenario would the `chevronElement` be undefined?

Reply:
I really shouldn’t. However, I did hit this on accident because I had forgotten to add a check for if the animation had fired when the element was destroyed (Angular just does that). 

I added a check to the animation handler so it doesn’t call this code in that case now, but I left the null check here just as a safety net since this is a public function that could be called with `undefined`.

Reply:
I ignored the branch because there is no current path that leads to `undefined` being passed in and we don’t have any tests that call adapter methods directly.

Reply:
I'd either:
1. Remove the `undefined` check, or
2. Make the method's parameter `ElementRef | undefined`.

I'd personally prefer the first option, and add a check for undefined later when a specific scenario is encountered that can be unit tested.